docs/requirements.txt

```diff
@@ -0,0 +1,11 @@
+autodocsumm==0.2.7
```

Comment:
rather than do this, should we add a tool to run pip freeze on ci-gpu and then use this as a constraints file to locally install these deps? my concern is that adding a requirements.txt with pins may drift.

Reply:
Removed the requirements.txt and added 2 building flows, one that uses a short script and relies on ci-gpu and one that just uses it to generate the list of dependencies. For now this works OK but the makefile + bash setup doesn't seem tenable for the larger goals, we might want to consider a Python runner similar to Rust's [`x.py`](https://github.com/rust-lang/rust/tree/master/src/bootstrap)